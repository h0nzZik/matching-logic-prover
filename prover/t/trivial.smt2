(assert ( = 1 1 ))
(assert (not ( = 2 2 )))
(check-sat)
