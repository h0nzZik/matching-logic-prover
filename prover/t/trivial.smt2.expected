<prover>
  <k>
    \exists { .Patterns } \and ( \not ( \equals ( 2 , 2 ) ) , \equals ( 1 , 1 ) , .Patterns ) ~> .SMTLIB2Script
  </k>
  <goals>
    <goal>
      <id>
        root
      </id>
      <active>
        true
      </active>
      <parent>
        .
      </parent>
      <claim>
        \forall { .Patterns } \or ( \equals ( 2 , 2 ) , \not ( \equals ( 1 , 1 ) ) , .Patterns )
      </claim>
      <strategy>
        success
      </strategy>
      <trace>
        .
      </trace>
    </goal>
  </goals>
  <declarations>
    .DeclarationCellSet
  </declarations>
</prover>
